### PR TITLE
[17.0][MIG] crm_lead_vat: Migration to 17.0

### PR DESCRIPTION
**Pull request by peluko00**

### CODE REVIEW

**Comment by peluko00** [comment]
Module migrated to version 17.0

cc https://github.com/APSL 148474

@miquelalzanillas @lbarry-apsl @javierobcn @samoros1 please review

**Comment by miquelalzanillas** [comment]
Working!

**Comment by samoros1** [comment]
Functionally reviewed working as expected!

**Review by miquelalzanillas — APPROVED** [review]

File: crm_lead_vat/tests/test_lead.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 class LeadCase(TransactionCase):
     def setUp(self):
```
Comment (pedrobaeza) [inline crm_lead_vat/tests/test_lead.py]: Please use `setUpClass`

**Review by samoros1 — APPROVED** [review]
Reviewed. Functionally working as expected.

File: crm_lead_vat/tests/test_lead.py (hunk 1 above)
Comment (peluko00) [reply crm_lead_vat/tests/test_lead.py]: Fix it!

**Review by javierobcn — APPROVED** [review]
Working

**Pull request merged**